### PR TITLE
use 0.8.19

### PR DESCRIPTION
**Pull request by Ferret-san**

### CODE REVIEW

File: foundry.toml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 [profile.default]
-solc_version = "0.8.22"
+solc_version = "0.8.19"
```
Comment (rach-id) [inline foundry.toml:2]: let's not downgrade the version of the compiler, only the contracts. Also, would need to investigate the optimisations that we removed because they're applied automatically in the new solidity version, whether we need to re-introduce them.

also, would be good to have more information on why this change is needed

File: src/Blobstream.sol [hunk 2]
```diff
@@ -36,7 +36,12 @@ struct Signature {
 /// block headers.
 /// @dev DO NOT REMOVE INHERITANCE OF THE FOLLOWING CONTRACTS: Initializable, UUPSUpgradeable and
 /// OwnableUpgradeable! They're essential for upgradability.
-contract Blobstream is IDAOracle, Initializable, UUPSUpgradeable, OwnableUpgradeable {
+contract Blobstream is
```
Comment (rach-id) [inline src/Blobstream.sol:39]: let's not format and downgrade the version in the same PR.

Also, which formatter did you use to have all of these changes? ideally, we have that in the CI to enforce a certain formatting.

**Comment by Ferret-san** [comment]
ah mb @rach-id, I accidentally opened this branch against main and not my own fork. Plz feel free to ignore
